akka-docs/rst/scala/persistence.rst

```diff
@@ -634,10 +634,7 @@ indeed adapt it return the adapted event(s) for it, other adapters which do not
 adaptation simply return ``EventSeq.empty``. The adapted events are then delivered in-order to the ``PersistentActor`` during replay.
 
 .. note::
```

Comment:
higher up in this section there is a note: "Complete documentation featuring use-cases and implementation examples for this feature will follow shortly."

Should that be removed now?